tests/test_classifiers/test_sparse_sklearn_intent_classifier.py

```diff
@@ -0,0 +1,43 @@
+import pathlib
+
+from rasa.train import train_nlu
+from rasa.cli.utils import get_validated_path
+from rasa.model import get_model, get_model_subdirectories
+from rasa.core.interpreter import RasaNLUInterpreter
+from rasa.shared.nlu.training_data.loading import load_data
+
+
+def load_interpreter(model_path):
+    model_path = pathlib.Path(model_path)
+    model_path_val = get_validated_path(str(model_path), "model")
+    model_path = get_model(model_path_val)
+    _, nlu_model = get_model_subdirectories(model_path)
+    return RasaNLUInterpreter(nlu_model)
+
+
+def test_predict():
+    NLU_DATA_PATH = "tests/data/nlu/en/nlu.md"
+    model_path = train_nlu(
+        nlu_data=NLU_DATA_PATH,
+        config="tests/configs/sparse-sklearn-intent-classifier-config.yml",
+        output="models",
+    )
+
```

Comment:
Could you also add a smoketest? 

https://github.com/RasaHQ/rasa-nlu-examples/blob/master/tests/test_smoke/test_english.py#L15